### ISSUE TITLE
Client Crash from Chunk generation, snow layer calculator 

### ISSUE THREAD

**Issue by WaxilliumGlass** [Team-RTG/Realistic-Terrain-Generation#1012]
##### Affected Mod Versions:
- Forge version:12.18.2.2109
- RTG version:4.1.1.4
- Pastebin includes mod list
##### Issue Description:

Client Crash when generating new chunks. Crash log seems to indicate that its a problem with RTG figuring out snow layers for BoP's Cold Desert, potentially other snowy biomes. Tested by loading world again and got same crash again on attempt to generate chunks. 

Crash log:
http://pastebin.com/b0uyxz4v

Fml-client-latest: not sure if you guys need this or not but here it is
https://gist.github.com/WaxilliumGlass/1be95556c910bc884217a9a03267dfe6

**Comment by srs-bsns** [comment]
https://github.com/Team-RTG/Realistic-Terrain-Generation/blob/1.10.2-dev/src/main/java/rtg/util/SnowHeightCalculator.java#L10

``` java
byte h = (byte) ((noise[x * 16 + z] - ((int) noise[x * 16 + z])) * 8);
```

This needs extra checking to be sure `h` is locked into a range of `1-8`.

**Comment by lightningo7** [comment]
https://github.com/Team-RTG/Realistic-Terrain-Generation/blob/1.10.2-dev/src/main/java/rtg/util/SnowHeightCalculator.java#L12-L14

If you look at that is it being locked into a 1-8 range.

**Comment by whichonespink44** [comment]
Fixed in 616a7cf
